File: check_db.js
const { createClient } = require('@libsql/client');
const client = createClient({ url: 'file:C:/Users/Admin/Desktop/My Project/Slip And Receipt/All fixed/TapowanPublicSchool-fixed/server/school.db' });
async function run() {
  const tables = await client.execute("SELECT name FROM sqlite_master WHERE type='table'");
  for (const row of tables.rows) {
    const tableName = row.name;
    const count = await client.execute(`SELECT count(*) as c FROM ${tableName}`);
    console.log(tableName, count.rows[0].c);
  }
}
run().catch(console.error);

File: check_db.py
import sqlite3
import os

paths = [
  os.path.join(os.path.expanduser('~'), 'AppData', 'Roaming', 'school-management-system', 'school.db'),
  os.path.join(os.path.expanduser('~'), 'AppData', 'Roaming', 'Tapowan Public School System', 'school.db'),
  os.path.join(os.path.expanduser('~'), 'Desktop', 'My Project', 'Slip & Receipt', 'All fixed', 'TapowanPublicSchool-fixed', 'server', 'school.db')
]

for p in paths:
    if os.path.exists(p):
        print("Found DB:", p)
        conn = sqlite3.connect(p)
        print("Tables:", conn.execute("SELECT name FROM sqlite_master WHERE type='table';").fetchall())
        conn.close()
